src/go/mongolib/util/main_test.go

```diff
@@ -145,9 +145,7 @@ func TestGetReplicasetMembers(t *testing.T) {
 	database.EXPECT().Run(bson.D{{"serverStatus", 1}, {"recordStats", 1}}, gomock.Any()).SetArg(1, ss)
 	session.EXPECT().Close()
 
-	session.EXPECT().Close()
```

Comment:
This test were broken before my changes, I fixed them. It looks like `runtests.sh` script is broken, I didn't fixed it. https://github.com/percona/percona-toolkit/blob/3.0/src/go/runtests.sh#L7-L12 Script lists only top level packages so e.g. `mongolib` but not subpackages e.g. `mongolib/util` and then it runs tests only for that package with `go test` (alternative would be `go test ./...` but this doesn't work for coverage).
travis.ci would be nice to have too :P